azure-pipelines/build-all.yml

```diff
@@ -2,24 +2,94 @@ parameters:
 - name: versionNumberOverride
   type: string
   default: 'default'
-
+- name: isOfficial
+  type: boolean
+- name: channel
+  values:
+  - release
+  - prerelease
+  - auto
+  default: auto
 stages:
 - stage: Build
   displayName: 'Build VSIXs'
   dependsOn: []
   jobs:
-  - template: build.yml
+  - job: SetChannelVariable
+    displayName: 'Set Channel Variable'
+    pool:
+      ${{ if eq(parameters.isOfficial, true) }}:
+        name: netcore1espool-internal
+        image: 1es-ubuntu-2204-pt
+      ${{ else }}:
+        name: Azure Pipelines
+        vmImage: ubuntu-latest
+      os: linux
+    steps:
+    - pwsh: |
+        $isPrerelease = $true
```

Comment:
was having some issues with uploading both release and prerelease artifacts.  Decided to change it slightly to instead figure out release or prerelease based on the branch, with an optional override.

If we need a release build from not the release branch, its easy to queue up another build with the channel override set.